src/test/java/core/basesyntax/service/impl/ReportServiceImplTest.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.ReportServiceImpl;
+import core.basesyntax.dao.impl.FruitDao;
+import core.basesyntax.dao.impl.FruitDaoImpl;
+import core.basesyntax.db.Storage;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class ReportServiceImplTest {
+    private static final String APPLE = "apple";
+    private static final String BANANA = "banana";
```

Comment:
The same

Reply:
Or you can create some Utill class(FruitTestConstants in package util) for saving all using test constants

Reply:
done, thank you